lib/galaxy/managers/group_roles.py

```diff
@@ -46,8 +39,8 @@ def show(self, trans: ProvidesAppContext, id: EncodedDatabaseIdField, group_id:
         group_role = self._get_group_role(trans, group, role)
         if group_role is None:
             raise ObjectNotFound(f"Role {role.name} not in group {group.name}")
-
-        return self._serialize_group_role(group_id, role)
+        return role
```

Comment:
```suggestion
        return group_role
```

Reply:
hmm... Or are you returning role after verifying that an association between this role and this group exists? If so, then leave it as is - the logic is correct, although the naming makes it confusing. If the sole purpose of `_get_group_role()` is to verify that an association between a group and a role exists, then it might be more appropriate if it returned true or false, and was named accordingly. That would be my suggestion, but that's beyond the scope of this PR.

Reply:
Returning role on these was bothering me as well, but it is needed in the serialization in the next step, and this was the cleanest way to get it there. 

I agree with your suggestion of returning true and false, however we would need a separate function for that since we actually need` _get_group_role() `as is in the `delete()` function

Reply:
Yes, I see. We can leave it as is.